src-web/nls/platform.properties

```diff
@@ -177,8 +177,9 @@ ansible.scheduleMode.disabled.title = Disable automation
 ansible.scheduleMode.disabled.info = Automation does not run until this setting is updated.
 ansible.unsaved.data = You made changes to the configuration. Do you want to save your policy automation? Click the Cancel button to close.
 ansible.loading.info = Loading policy violation automation data...
-ansible.create.success = New policy violation automation ${0} is successfully created on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
-ansible.edit.success = Existing policy violation automation ${0} is successfully updated on the namespace ${1}, now you can switch to History page to check ansible history information, or close this panel.
+ansible.manual.success = The automation has been manually triggered successfully. Please see history tabe to check ansible history information.
+ansible.once.success = Policy automation has been set to once mode. The automation will be automatically triggered once when policy is in NonCompliant status. Please see history tab to check ansible history information.
```

Comment:
Policy automation is set to once mode. The automation is automatically initiated once when the policy is in a NonCompliant state. View the History tab to check the ansible history information.